docs/apidocs/qiskit_nature.second_q.properties.s_operators.rst

```diff
@@ -0,0 +1,28 @@
+﻿
+
+s_operators
+=============================================
+
+.. automodule:: qiskit_nature.second_q.properties.s_operators
+
+   
```

Comment:
This file has a lot of blank lines - not that it really matters, but...

Reply:
I think some others do and am willing to ignore this since it has no effect in any case